src/main/java/net/quedex/api/user/OrderListener.java

```diff
@@ -12,6 +12,10 @@
 
     void onOrderCancelFailed(OrderCancelFailed orderCancelFailed);
 
+    void onAllOrdersCancelled();
+
+    void onCancelAllOrdersFailed(CancelAllOrdersFailedCause cause);
```

Comment:
I think it should be wrapped in `CancelAllOrdersFailed` - we won't be able to extend it easily, e.g. with per-instrument or per-instrument-class cancellation.

Reply:
OK